### PR TITLE
APPS-579 Add regression tests for expression evaluation: boolean, substitution.

### PR DESCRIPTION
**Pull request by r-i-v-a**

### CODE REVIEW

File: scripts/clean_build.sh [hunk 1]
```diff
@@ -11,9 +11,7 @@ rm dx*.jar
 # Clean artifacts from prev. builds, platform
 username=`dx whoami --id`
 dx rm -r dxCompiler_playground:/builds/$username
-dx rm -r dxCompiler_playground:/builds/user-$username
```
Comment (r-i-v-a) [inline scripts/clean_build.sh:14]: `dx whoami` with `--id` already returns the user-xxxx form of the username.

Looking at the implementation of dxApi.whoami(), it looks consistent with the dxpy one (both use the same system API call), so the unit_tests folder should also get the same name.

File: test/bugs/apps_579_boolean_flag_expr.wdl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+version 1.1
```
Comment (r-i-v-a) [inline test/bugs/apps_579_boolean_flag_expr.wdl:1]: @jdidion   Should I actually make these 1.0 tests? Do we only test against 1.1 for new features not existing in 1.0?

Comment (r-i-v-a) [reply test/bugs/apps_579_boolean_flag_expr.wdl:1]: From comment on other PR -- 1.1 is ok

**Comment by r-i-v-a** [comment]
This one is ready for review @jdidion

**Review by jdidion — APPROVED** [review]

**Pull request merged**